### ISSUE TITLE
v1.3.0 - failed to extract a Datadog span request: [error code 11]

### ISSUE THREAD

**Issue by afares-hims** [DataDog/nginx-datadog#116]
Hello,

We're having this error when using v1.3.0 (nginx v1.25.5), traces are coming correctly, though:

```
2024/09/23 17:35:50 [error] 19#19: *4862 failed to extract a Datadog span request 000056033FAEA0C0: [error code 11]: While extracting trace context, an error occurred: There's neither a trace ID nor a parent span ID to extract., client: x.x.x.x, server: localhost, request: "GET /foo/bar HTTP/1.1", host: "api.domain.com"
```

Can you help with this, please?
Thank you!

**Comment by dmehala** [comment]
Hey @afares-hims 

Thank you for reporting this issue. It's an harmless error log usually happening when there is no tracing context to extract. The frequency depends of how often the server start the trace or not.

TL;DR: The log can be ignored. I can understand it's verbose, so I am working on a quick fix and will probably release the quick fix soon.
